test/python/circuit/classical/test_expr_constructors.py

```diff
@@ -610,3 +621,311 @@ def test_shift_forbidden(self, function):
             function(Duration.dt(1000), 1)
         with self.assertRaisesRegex(TypeError, "invalid types"):
             function(Duration.dt(1000), Duration.dt(1000))
+
+    @ddt.data(
+        (expr.add, expr.Binary.Op.ADD),
+        (expr.sub, expr.Binary.Op.SUB),
+    )
+    @ddt.unpack
+    def test_binary_sum_explicit(self, function, opcode):
+        cr = ClassicalRegister(8, "c")
+
+        self.assertEqual(
+            function(cr, 200),
+            expr.Binary(
+                opcode, expr.Var(cr, types.Uint(8)), expr.Value(200, types.Uint(8)), types.Uint(8)
+            ),
+        )
+        self.assertFalse(function(cr, 200).const)
```

Comment:
Super minor, but the `expr.Binary` equality check should already be doing the `const` checks.  Could we have written the tests such that the `const` is set explicitly in the `expr.Binary` constructor?

(No need to change all the tests now.)